database scripts/Create_main_tables.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE Owners (
+	email varchar(128) not null primary key, 
+    fname varchar(64), 
+    lname varchar(64), 
+    address varchar(128));
```

Comment:
Would it maybe be a good idea to split address into 
street name and number
zip code
?